buildpack/runtime_components/storage.py

```diff
@@ -91,21 +92,18 @@ def _get_s3_specific_config(vcap_services, m2ee):
         tvm_endpoint
         and tvm_username
         and tvm_password
-        and m2ee.config.get_runtime_version() < 9.2
-    ):
-        logging.info(
-            "S3 TVM config detected, fetching IAM credentials from TVM"
-        )
-        access_key, secret = _get_credentials_from_tvm(
-            tvm_endpoint, tvm_username, tvm_password
+        and (
+            m2ee.config.get_runtime_version() >= 9.2
```

Comment:
As an alternative, you can use `from buildpack import runtime` and `runtime.get_version()` to get the version. This will not require the m2ee client and get the version directly from metadata.

Reply:
Thanks for your feedback! Done :+1:

Reply:
Sorry @markbastiaans I just reverted it again. It seems `runtime.get_version()` requires an argument (build_path).

Turning:
```
m2ee.config.get_runtime_version()
```
into:
```
runtime.get_version(os.path.abspath("."))
```
doesn't make it more readable to me. Can the build_path be autodetected? `m2ee.config.get_runtime_version()` also doesn't require a build_path argument.

Reply:
Hi @pommi - I have an MR that will automate this. You can leave the code as-is and we can change it in the future.